grid-proxy/tests/queries/utils.go

```diff
@@ -59,3 +59,12 @@ func SerializeFilter[F Filter](f F) string {
 
 	return res
 }
+
+func getRandomSlice[T any](slice []T) []T {
+	randomLen := rand.Intn(len(slice))
```

Comment:
I don't see a rand.Seed call?

Reply:
found it here. it is based on a flag
https://github.com/threefoldtech/tfgrid-sdk-go/blob/3358c61e65e67e157b5ce497ceb9799b86c101da/grid-proxy/tests/queries/main_test.go#L60